swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from swap_meet.item import Item
+class Clothing(Item):
+    def __init__(self, condition = 0):
+        Item.__init__(self, "Clothing", condition )
+    def __str__(self):
+        return "The finest clothing you could wear."
```

Comment:
Nice use of inheritance here. It's preferable to use the super() syntax instead, so you can rewrite this like so:

```suggestion
        super().__init__(self, "Clothing", condition )
        
    def __str__(self):
        return "The finest clothing you could wear."
```

Added an extra space just because it's easier to read code when there are spaces in between methods/functions.

Reply:
^ The above can also be applied to Decor and Electronics